test/shutdown_SUITE.erl

```diff
@@ -468,7 +526,14 @@ http2_server_goaway_many_streams(_) ->
 	StreamRef1 = gun:get(ConnPid, "/"),
 	StreamRef2 = gun:get(ConnPid, "/"),
 	StreamRef3 = gun:get(ConnPid, "/"),
+	StreamRef4 = gun:get(ConnPid, "/"),
 	ConnRef = monitor(process, ConnPid),
+	%% GOAWAY received. Stream 4 is cancelled.
+	{gun_error, ConnPid, StreamRef4, Reason4} = receive E4 -> E4 end,
+	{goaway, no_error, _} = Reason4,
+	timer:sleep(100),
```

Comment:
I don't think this timer:sleep is necessary.

Reply:
You're right. Deleted.